spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/annotation/ContinueSpan.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2013-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.sleuth.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Tells Sleuth that all Sleuth related annotations should be applied
+ * to an existing span instead of creating a new one.
+ *
+ * @author Marcin Grzejszczak
+ * @since 1.2.0
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Inherited
+@Target(value = { ElementType.METHOD, ElementType.TYPE })
+public @interface ContinueSpan {
+
+	/**
+	 * The value passed to the annotation will be used and the framework
+	 * will create two events with the {@code .start} and {@code .end} suffixes
+	 */
+	String log() default "";
```

Comment:
hmm.. I kindof prefer start/end, as that allows you to emulate existing stuff. ex in finagle there are 3 parameters to a trace filter

https://github.com/twitter/finagle/blob/develop/finagle-core/src/main/scala/com/twitter/finagle/tracing/TraceInitializerFilter.scala#L132
```
before: Annotation <- added at the beginning of a task
after: Annotation <- added unconditionally after a task
afterFailure: Annotation + ": $error.class: $error.message" <- additionally added when the task failed
```